README.md

```diff
@@ -78,11 +78,11 @@ smb, auth, vfs, oplock, ipc, conn, rdma
 
 - ksmbd.adduser
   - Adds (-a), updates (-u), or deletes (-d) a user from user database.
-  - Default database file is `/etc/ksmbd/users.db`
+  - Default database file is `/usr/local/etc/ksmbd/users.db`
 
 - ksmbd.addshare
   - Adds (-a), updates (-u), or deletes (-d) a net share from config file.
-  - Default config file is `/etc/ksmbd/smb.conf`
+  - Default config file is `/usr/local/etc/ksmbd/smb.conf`
```

Comment:
/use/local/etc ... Never seen that before.

Reply:
The default prefix for autotools and meson is /usr/local. It then follows that the default sysconfdir for both is /usr/local/etc. When Linux distributions' packaging scripts invoke autotooled configure, they typically set prefix to /usr and sysconfdir to /etc. If they do not set sysconfdir (and this situation is not handled as a special case), it would default to /usr/etc, which is practically never used. Meson is clever regarding this as it handles the /usr prefix as a special case and implicitly sets sysconfdir to /etc. I suggest you look into this some further, you'll be surprised to learn that this is the norm.

Reply:
So for /usr this would evaluate to /usr/etc. I hate this change.

Reply:
meh fair enough.